influxql/parser_test.go

```diff
@@ -81,6 +81,21 @@ func TestParser_ParseQuery_NoSemicolon(t *testing.T) {
 	}
 }
 
+func TestParser_ParseBoundParameter(t *testing.T) {
```

Comment:
If you can change this test name to `SQLInjection`, I would like that more. We already test bound parameters in another section and this is a test for sql injection.